src/libawkward/array/RegularArray.cpp

```diff
@@ -1095,7 +1095,7 @@ namespace awkward {
       Index64 nextadvanced(len*nextsize);
 
       struct Error err =
-        awkward_regulararray_getitem_next_range_spreadadvanced_64(
+        kernel::regulararray_getitem_next_range_spreadadvanced<int64_t>(
```

Comment:
Don't: the kernel name is `awkward_regulararray_getitem_next_range_spreadadvanced_64`.